webfront/searcher/elastic_controller.py

```diff
@@ -343,6 +343,8 @@ def get_list_of_endpoint(self, endpoint, query=None, rows=10, start=0, cursor=No
             for x in response["aggregations"]["groups"]["buckets"]
         ]
         if reset_direction:
+            if len(facet["aggs"]["groups"]["composite"]["sources"]) > 2:
+                accessions.reverse()
```

Comment:
I _think_ that `accessions.reverse()` could be called regardless of the number of sorting fields.

If printing `accessions` (not reordered) for `/api/protein/uniprot/entry/InterPro/IPR000030/?page_size=3`:

```py
print(accessions)
['a0a024jpi2', 'a0a024jqd1', 'a0a024jqv5']
```

Then, following the `next` page:
```py
print(accessions)
['a0a024jr82', 'a0a024jrb0', 'a0a024jrv9']
```

Finally, going back to first page (`previous`):

```py
print(accessions)
['a0a024jqv5', 'a0a024jqd1', 'a0a024jpi2']
```

Same accessions as in the first request, but in the opposite order.

Reply:
Yeah, I think you are right, I put that because I noticed a test failing, but seems that my tests were failing locally for a different reason. I'll push this change once my local tests finish running, and if everything runs in GH actions I'll merge